### PR TITLE
Print missing metrics to log if they were not computed

### PR DESCRIPTION
**Pull request by fridex**

### CODE REVIEW

File: wsgi.py [hunk 1]
```diff
@@ -141,9 +141,10 @@ def metrics():
     if not _INITIALIZED:
         with _INITIALIZED_LOCK:
             _LOGGER.warning(
-                "Not all metrics were gathered, the service is not ready yet (%d/%d)",
+                "Not all metrics were gathered, the service is not ready yet (%d/%d), missing: %s",
                 sum(_EXECUTED.values()),
                 len(REGISTERED_JOBS),
+                [k for k, v in _EXECUTED if v == 0],
```
Comment (pacospace) [inline wsgi.py]: missing `.items()`?

Comment (fridex) [reply wsgi.py]: Nice catch! :+1:

Comment (pacospace) [reply wsgi.py]: But late :/

Comment (fridex) [reply wsgi.py]: Better late than never (or in prod :) )

**Pull request merged**